Flick/Models/FriendRequest.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  FriendRequest.swift
+//  Flick
+//
+//  Created by Lucy Xu on 1/8/21.
+//  Copyright © 2021 flick. All rights reserved.
+//
+
+struct FriendRequest: Codable {
+    var fromUser: UserProfile
+    var created: String
+    var rejected: Bool?
+    var viewed: Bool?
```

Comment:
should we just take it out?

Reply:
hmm maybe I will just leave in for now since idk what backend will do or if we might want this later?